app/backends.py

```diff
@@ -19,32 +19,28 @@ def configure_user(self, user: User) -> User:
 
         :returns: [User] The user object with updates to is_staff, email, first_name, last_name fields.
         """
-        url = "https://yalies.io/api/people"
-        querystring = {"query": user.username}
+        url = 'https://yalies.io/api/people'
+        payload = {
+            'filters': {
+                'netid': user.username,
+            }
+        }
         headers = {
-            'Authorization': f"Bearer {get_secret('YALIES_API')}",
+            'Authorization': 'Bearer ' + get_secret('YALIES_API'),
         }
 
         try:
-            response = requests.post(url, headers=headers, json=querystring)
+            response = requests.post(url, headers=headers, json=payload)
             response.raise_for_status()
             data = response.json()[0]
```

Comment:
It may be a good idea to handle the case where no results are returned. In theory it should be a rare occurrence, but it can happen.

Reply:
Great point, currently the default behavior is to conditionally render any info we get from the response (since we still have the netID if nothing is returned). Later on, after refactoring a lot of the repo, this is a perfect spot to indicate the user needs to manually enter info.